packages/apps/proof-of-us/src/components/List/ListItem.tsx

```diff
@@ -28,15 +29,18 @@ interface ITempToken extends Token {
 
 export const ListItem: FC<IProps> = ({ token }) => {
   //@todo fix the tokenURI. it is now missing from the graph
-  const uri =
-    (token as ITempToken).info?.uri ??
-    'https://bafybeiemmkua6swmnvx4toqnhhmqavqkm4z5zltkiuj4yuq3kwnm576esq.ipfs.nftstorage.link/metadata';
+  const uri = (token as ITempToken).info?.uri;
   const { data, isLoading } = useSWR(uri, fetchManifestData, {
     revalidateOnFocus: false,
   });
 
   return (
-    <li className={listItemClass}>
+    <motion.li
+      className={listItemClass}
+      initial={{ opacity: 0, left: '500px' }}
```

Comment:
If possible, it's better to use transforms so the animations are you accelerated. You can try

```suggestion
      initial={{ opacity: 0, x: '500px' }}
```

Reply:
you are correct of course.